homeassistant/components/modbus/__init__.py

```diff
@@ -55,7 +55,6 @@
     CALL_TYPE_DISCRETE,
     CALL_TYPE_REGISTER_HOLDING,
     CALL_TYPE_REGISTER_INPUT,
-    CALL_TYPE_WRITE_REGISTER,
```

Comment:
This was also still imported at https://github.com/home-assistant/core/blob/dev/homeassistant/components/flexit/climate.py

Reply:
We should remove them from the other integrations